Package.swift

```diff
@@ -27,8 +27,8 @@ let package = Package(
     .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .exact("0.8.17"))
   ],
   targets: [
-    .target(name: "ApplicativeRouter", dependencies: ["Either", "Prelude"]),
-    .testTarget(name: "ApplicativeRouterTests", dependencies: ["ApplicativeRouter", "Optics"]),
+    .target(name: "ApplicativeRouter", dependencies: ["Either", "Optics", "Prelude"]),
```

Comment:
I could do without `Optics` in the router if we didn't wanna bring in this dependency. just hard to say no to optics!